apps/api-extractor/src/extractor/Extractor.ts

```diff
@@ -192,6 +206,10 @@ export class Extractor {
           this._absoluteRootFolder
         );
 
+        if (!commandLine.options.skipLibCheck && options.skipLibCheck) {
+          commandLine.options.skipLibCheck = true;
```

Comment:
>commandLine.options.skipLibCheck = true; [](start = 10, length = 40)

Should we print a log message that will help people when investigating API Extractor failures?

colors.cyan("API Extractor was invoked with --skip-lib-check. This is not recommended and may cause incorrect type analysis.")